app.js

```diff
@@ -119,7 +119,7 @@ if (sthingsEnabled) {
       if (err.constructor.name === 'ValidationError') {
         return res.status(STATUS_CODES.BAD_REQUEST).json({error: 'validation-error', keyPath: err.keyPath.join('.'), message: err.message});
       }
-      res.status(STATUS_CODES.INTERNAL_SERVER_ERROR).json({error: 'internal-server-error'});
+      return res.status(STATUS_CODES.INTERNAL_SERVER_ERROR).json({error: 'internal-server-error'});
```

Comment:
returns in those functions does not make sense. Please use if else statement instead. for example
 
```
   (req, res, next) => {
       if (req.isAuthenticated()) {
         return next();
       }
       return res.status(STATUS_CODES.UNAUTHORIZED).json({error: 'unauthorized'});
     },

```
====>

  ```
 (req, res, next) => {
      if (req.isAuthenticated()) {
        next();
      } else {
          res.status(STATUS_CODES.UNAUTHORIZED).json({error: 'unauthorized'});
      }
    },
```

Reply:
done